docs/usage/getting-started/running.md

```diff
@@ -1,70 +1,94 @@
+- provide the infrastructure that Renovate runs on,
+- provision Renovate's global config,
+- make sure Renovate bot runs regularly,
+- make sure Renovate bot itself is updated
 
-Read our section on [Self-hosting on Windows](./installing-onboarding.md#self-hosting-on-windows) to prevent line endings from confusing Renovate bot.
+If you're self-hosting Renovate on Windows, read [Self-hosting on Windows](./installing-onboarding.md#self-hosting-on-windows) to prevent line endings from confusing Renovate bot.
 
 ### Available distributions
 
 #### npm package (CLI)
 
 Renovate's Open Source CLI is built and distributed as the npm package `renovate`.
-You can run this directly in any Node.js environment - even via `npx` - and it will process all the repositories it is configured with, before exiting.
-When you install Renovate from npm it naturally does not come bundled with any third-party tools or languages such as Ruby, Python, Composer, Bundler, Poetry, etc.
+You can run this package in any Node.js environment - even via `npx` - and it will process all the repositories it is configured with, before exiting.
+
+Renovate's npm package does not bundle any third-party tools or languages like Ruby, Python, Composer, Bundler, Poetry, etc.
 If you need Renovate to support any non-npm lock files like Bundler then you'll need to make sure all required third-party tools are pre-installed in the same environment alongside Renovate before you run it.
 
 The `renovate` npm package is compatible with all of Renovate's supported platforms.
 
-Renovate requires Node.js `>=14.15.0` and Git `>=2.33.0`.
+Renovate requires Node.js `>=18.12.0` and Git `>=2.33.0`.
+
+#### Docker images
+
+Renovate is also distributed as Docker images on Docker Hub (`renovate/renovate`) and GitHub container registry (`ghcr.io/renovatebot/renovate`).
+These Docker images work on all the hosting platforms that Renovate supports.
+
+You must run the Docker images in a `linux/amd64` container.
+We're working on `linux/arm64` container support.
+You can't run the Docker images on a Windows or macOS container.
+
+In general, you can run Renovate on Windows with global mode.
+
+There are two flavors:
 
-#### Docker image
+- The `-slim` image
+- The default "full" image, this is the image you'll get if you use the `latest` tag
 
-The `renovate` npm package is also distributed via pre-built Node.js images on Docker Hub (`renovate/renovate`).
+##### The slim image
 
-The `slim` image contains only Node.js so works if either:
+The `-slim` image only comes with the Node.js environment.
+By default, the image installs the required tools when needed.
 
-- You do not require any additional package managers, or
-- You map the Docker socket into the container so that Renovate can dynamically invoke "sidecar" images when necessary
+If you want, you can map the Docker socket into the container so that Renovate can dynamically invoke "sidecar" images when needed.
```

Comment:
link to binary source? because that needs to be set to docker too

Reply:
```suggestion
If you want, you can map the Docker socket into the container so that Renovate can dynamically invoke "sidecar" images when needed.
You'll need to set `binarySource=docker` for this to work.
Read the [`binarySource` config option docs](https://docs.renovatebot.com/self-hosted-configuration/#binarysource) for more information.
```

Reply:
i think we can use a relative link here, otherwise LGTM